src/state/ducks/participants.js

```diff
@@ -64,7 +64,7 @@ const reducer = function(state = initialState, action) {
   switch (type) {
     case PARTICIPANT_JOINED: {
       // TODO: check initial values for audio/video
-      const participant = { ...payload, audio: true, video: true, stream_timestamp: null };
```

Comment:
So far, we need `audio` attribute to set set the `Mute` button. Which is the plan?

Reply:
the plan is that at that point of time, that value is not known, so I am not setting any value.

Reply:
So, fine with me. And for Travis now ;)